tests/test_environment_MatchEnv.py

```diff
@@ -204,6 +204,24 @@ def test_cluster_hardr(self):
             err_msg="BCC Cluster Environment fail",
         )
 
+    def _make_global_neighborlist(self, sys):
+        """Get neighborlist where all particles are neighbors."""
+        box, points = sys
```

Comment:
This function signature is strange. If it's intended to be called with an arbitrary system-like object, we should be calling `from_system` and then accessing the attributes. If it's always called with a `box, points` tuple (which seems to be the case) it should just accept them as separate parameters. The current approach communicates mixed messages about the intent.

Reply:
I just wrote this quick-and-dirty without thinking too much about the function signature since its just a helper method in a unit test, but I can update it to be clearer.